driver.py

```diff
@@ -47,19 +49,26 @@ def _ensure_image(testkit_path, docker_image_path, branch_name, driver_name,
 
     # This will use the driver folder as build context.
     docker.build_and_tag(image_name, docker_image_path,
-                         log_path=artifacts_path)
+                         log_path=artifacts_path, args=build_args)
 
     return image_name
 
 
+def _get_build_args():
+    return {k.removeprefix(BUILD_ARG_PREFIX): v
```

Comment:
`some_str.removeprefix` was added in Python version 3.9. TestKit currently is written to work with Python 3.7. Consider doing something ugly like `k[len(BUILD_ARG_PREFIX):]`